doc/sphinx/language/core/definitions.rst

```diff
@@ -121,9 +121,9 @@ Assertions and proofs
 ---------------------
 
 An assertion states a proposition (or a type) of which the proof (or an
-inhabitant of the type) is interactively built using tactics. The interactive
-proof mode is described in Chapter :ref:`proofhandling` and the tactics in
-Chapter :ref:`Tactics`. The basic assertion command is:
+inhabitant of the type) is interactively built using :term:`tactics <tactic>`.
+Proof mode is described in the :ref:`writing-proofs` chapter.  Tactics are
+described in the :ref:`Tactics` chapter. The basic assertion command is:
```

Comment:
There are tactics in the Automatic Solvers chapter, too.  I made it "Common tactics are in ...".  I thought the more-specific reference to writing-proofs was a plus.  (Also, BTW, the distinction between chapters, sections and subsections is murky.  How does the user know which is which?  Maybe it would help to add the work "Chapter" near top of each chapter.  Perhaps even use structured numbering like "1.2.1" for chapter 1, section 2, subsection 1.  Another topic for later.)

If you really want to combine them, I will do that.

Reply:
Looks perfect now, thanks!

> Also, BTW, the distinction between chapters, sections and subsections is murky. How does the user know which is which? Maybe it would help to add the work "Chapter" near top of each chapter. Perhaps even use structured numbering like "1.2.1" for chapter 1, section 2, subsection 1. Another topic for later.

Ultimately, I think we want to call "chapter" only the top-level ones, but because we have transformed historic chapters into sections it's not surprising that the language used to refer to them will fluctuate for a while. That being said, I thought we had a solution for this which was to write ``` see :ref:`foobar` ``` instead of ```see the :ref:`foobar` chapter``` or ```see the :ref:`foobar` section```.

Reply:
I see your point.  I prefer the phrasing "foo is described in :ref:" to "see :ref: for information on foo". mentioning the specific detail "foo" first" in the sentence.  We can review these reference phrases at a later date.